src/app/pages/PerpetualPage/components/TradeDialog/components/ConfirmationStep.tsx

```diff
@@ -204,6 +204,12 @@ const getTranslations = (
         count,
       },
     ),
-    text: <p>{t(translations.perpetualPage.processTrade.texts.confirm)}</p>,
+    text: (
+      <p>
+        {t(translations.perpetualPage.processTrade.texts.confirm, {
+          wallet: '',
```

Comment:
Please don't leave it empty. You should be able to get the wallet with the WalletContext or walletService